app/views/search/search.html.erb

```diff
@@ -1,5 +1,5 @@
 <% content_for :page_title do %>
-   <%= t('.results_page_title', :query => @query ) %>
+   <%= t('.results_page_title', :query => @topic ) %>
```

Comment:
We should introduce some extra test coverage so we're confident we have the correct behaviour for different contexts.

If you do a normal search, for example, https://find-data-pr-501.herokuapp.com/search?q=Data the title now says 'Results for "" - data.gov.uk"' because the topic is empty.

Reply:
👍